modules/enabling-intel-gaudi-ai-accelerators.adoc

```diff
@@ -4,36 +4,45 @@
 = Enabling Intel Gaudi AI accelerators
 
 [role='_abstract']
-Before you can use Intel Gaudi AI accelerators in {productname-short}, you must install the necessary dependencies and deploy the HabanaAI Operator. 
+Before you can use Intel Gaudi AI accelerators in {productname-short}, you must install the required dependencies, deploy the Intel Gaudi AI Accelerator Operator, and configure the environment.
 
 .Prerequisites
 ifdef::upstream,self-managed[]
 * You have logged in to {openshift-platform}.
 * You have the `cluster-admin` role in {openshift-platform}.
+* Intel Gaudi devices are installed and detected in your environment.
```

Comment:
Is there a way to rewrite to make parallel or would that make it more confusing/wordy? Something like this perhaps?:
"You have installed your Intel Gaudi accelerator and confirmed that it is detected in your environment."

Reply:
Changed to your suggestion.